nf_core/pipeline-template/modules.json

```
@@ -1,12 +1,14 @@
 {
     "name": "{{ name }}",
     "homePage": "https://github.com/{{ name }}",
-    "modules": {
-        "fastqc": {
-            "git_sha": "95e02f913fa0917fafae00e96e631d5b25a965f3"
-        },
-        "multiqc": {
-            "git_sha": "95e02f913fa0917fafae00e96e631d5b25a965f3"
+    "repos": {
+        "nf-core/modules": {
+            "fastqc": {
+                "git_sha": "95e02f913fa0917fafae00e96e631d5b25a965f3"
+            },
+            "multiqc": {
+                "git_sha": "95e02f913fa0917fafae00e96e631d5b25a965f3"
+            }
```

These old commit SHAs will not make the tests fail, but it might break `nf-core modules list` and perhaps other things also, so I think we should update these as well. 
```suggestion
    "repos": {
        "nf-core/modules": {
            "fastqc": {
                "git_sha": "e937c7950af70930d1f34bb961403d9d2aa81c7d"
            },
            "multiqc": {
                "git_sha": "e937c7950af70930d1f34bb961403d9d2aa81c7d"
            }
```